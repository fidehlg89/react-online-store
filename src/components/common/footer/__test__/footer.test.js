import React from "react";
import ReactDOM from "react-dom";
import Footer from "..";

import { render, cleanup } from "@testing-library/react";
import renderer from "react-test-renderer"

afterEach(cleanup);

it("renders with out crashing", () => {
    const div=document.createElement('div');
    ReactDOM.render(<Footer></Footer>, div);    
});

it("matches snapshot", ()=>{
    const tree=renderer.create(<Footer></Footer>).toJSON();
    expect(tree).toMatchSnapshot();
});


