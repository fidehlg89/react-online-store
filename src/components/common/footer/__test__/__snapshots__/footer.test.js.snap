// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<footer
  className="sc-jRQBWg fGnSGf"
/>
`;
